src/test/java/CarTest.java

```diff
@@ -0,0 +1,45 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import io.suhan.racingcar.Car;
+import io.suhan.racingcar.generator.FixedNumberGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CarTest {
+    @Test
+    @DisplayName("자동차는 이름을 가지고 있다.")
+    void carHasName() {
+        String name = "neo";
```

Comment:
이거는 나중에 `영어로도 메서드명을 적어야하고, 한글로도 적어야하고 하나만 적을 수 있는 방법이 없나?`라는 고민이 들 수도 있는데요.

영어만 쓰기엔 나중에 별로 같다 싶으면 아래처럼 한글만 적는 방법도 있으니 참고하시길 바랍니다

```suggestion

@SuppressWarnings("NonAsciiCharacters")
@DisplayNameGeneration(ReplaceUnderscores.class)
public class CarTest {
    @Test
    void 자동차는_이름을_가지고_있다() {
        String name = "neo";
```

Reply:
안그래도 테스트 메소드명 짓기가 번거로웠는데 좋은 방법 소개해주셔서 감사합니다. 바로 적용했습니다~!